dask/dataframe/core.py

```diff
@@ -5073,21 +5073,23 @@ def set_index(
         npartitions: int | Literal["auto"] | None = None,
         divisions: Sequence | None = None,
         inplace: bool = False,
+        sort: bool = True,
         **kwargs,
     ):
         """Set the DataFrame index (row labels) using an existing column.
 
-        This realigns the dataset to be sorted by a new column. This can have a
+        If ``sort=False``, this function operates exactly like ``pandas.set_index``
+        and sets the index on the dataframe. If ``sort=True`` (default),
+        this function also sorts the dataframe by the new index. This can have a
```

Comment:
Nit
```suggestion
        and sets the index on the DataFrame. If ``sort=True`` (default),
        this function also sorts the DataFrame by the new index. This can have a
```